components/Common/Search/States/WithSearchResult.tsx

```diff
@@ -0,0 +1,44 @@
+import type { Result } from '@orama/orama';
+import NextLink from 'next/link';
+import type { FC } from 'react';
+
+import type { SearchDoc } from '@/components/Common/Search/States/WithSearchBox';
+import { pathToBreadcrumbs } from '@/components/Common/Search/utils';
+import { highlighter } from '@/next.orama.mjs';
+
+import styles from './index.module.css';
+
+type SearchResultProps = {
+  hit: Result<SearchDoc>;
+  searchTerm: string;
+};
+
+export const WithSearchResult: FC<SearchResultProps> = props => {
+  const isAPIResult = props.hit.document.siteSection.toLowerCase() === 'api';
+  const basePath = isAPIResult ? 'https://nodejs.org/docs/latest' : '/en';
+  const path = `${basePath}/${props.hit.document.path}`;
+
+  return (
+    <NextLink
+      key={props.hit.id}
+      href={path}
+      className={styles.fulltextSearchResult}
+      target={isAPIResult ? '_blank' : undefined}
```

Comment:
Why target blank if API result?

Reply:
Oh, right, while in development I push everything to https://nodejs.org/docs/latest/api. Do we have the API pages in the redesign already?

Reply:
No? API pages do not belong on the Node.js Website. Also latest API docs is just https://nodejs.org/api/ (that's the shortest way)

Reply:
We just don't need target at all nor rel attributes if you use the Link component.

Reply:
So, to clarify, what's the expected behavior when a user clicks on an `api` search result? They should get redirected to https://nodejs.org/api/, right? In a new tab or same window?

Reply:
Redirected, yes, same window